fix_includes.py

```diff
@@ -968,24 +987,43 @@ class Myclass;
   start_line = 0
   while start_line < len(file_lines):
     line_info = file_lines[start_line]
-    if line_info.deleted or line_info.type != _NAMESPACE_START_RE:
+    if (line_info.deleted or 
+        (line_info.type != _NAMESPACE_START_RE and
+        line_info.type != _NAMESPACE_START_ALLMAN_RE and
```

Comment:
These two lines should be indented one more space to align with the `line_info` above